Tests/ComposableArchitectureTests/TestStoreFailureTests.swift

```diff
@@ -1,294 +1,295 @@
-#if DEBUG
-  import ComposableArchitecture
-  import XCTest
-
-  @MainActor
-  final class TestStoreFailureTests: XCTestCase {
-    func testNoStateChangeFailure() {
-      enum Action { case first, second }
-      let store = TestStore(
-        initialState: 0,
-        reducer: Reduce<Int, Action> { state, action in
-          switch action {
-          case .first: return .init(value: .second)
-          case .second: return .none
-          }
+// no XCTExpectFailure on windows yet
+#if DEBUG && !os(Windows)
```

Comment:
When do you decide to opt out Windows entirely vs `XCTSkipIfWindowsExpectFailure`?

Reply:
I think these test might have been edited at two different times, one before XCTSkipIfWindowsExpectFailure was written and one after. If an entire file is running tests with expected failures it seems silly to skip them all individually, and rather just make a smaller change by running them only on non windows. Let me see if I can make this more aligned.

Reply:
looks like all of the tests in this file rely on XCTExpectFailure so I'm going to leave the entire file compiled out for Windows.